api/migrations/schema/Version20231111000042.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20231111000042 extends AbstractMigration {
+    public function getDescription(): string {
+        return 'Add DefaultLayout content type';
+    }
+
+    public function up(Schema $schema): void {
+        $this->addSql('INSERT INTO public.content_type (id, name, active, entityclass, jsonconfig, createtime, updatetime) VALUES (\'a4211c112939\', \'DefaultLayout\', true, \'App\Entity\ContentNode\DefaultLayout\', \'{"items":[{"slot":"main"},{"slot":"aside-top"},{"slot":"aside-bottom"}]}\', \'2023-11-11 00:00:42\', \'2023-11-11 00:00:42\');');
```

Comment:
It would be nice if you could split this up into 2 lines, else the line is very long.
e.g. like in Version20211207143737